docs/general/source/development_plan.rst

```diff
@@ -0,0 +1,225 @@
+        * PyNWB API for NWB:N: http://pynwb.readthedocs.io
+        * NWB:N data format specification: http://nwb-schema.readthedocs.io
+        * Specification Language: http://schema-language.readthedocs.io
+        * Data Storage: http://nwb-storage.readthedocs.io
+
+    * **Format Documentation Tools** To ensure consistency between the NWB:N format specification and documentation we have developed a set of tools generate Sphinx RST documents from the YAML specification sources. The tools are available in `docs/utils` as part of the nwb-schema repository for Python 2/3.
+
+    * **Main contributors for this release** :
+        * Oliver Ruebel and Andrew Tritt (LBNL) : Documentation leads
+        * Several of the documents have been ported from NWB:N 1.x, which were originally created by Jeff Teeters (UCB) et al.
+        * Other teams have also contributed bug fixes
+
+* **MatNWB** MatNWB is a Matlab API for NWB:N. MatNWB generates Matlab classes for representing NWB:N neurodata_type directly from the YAML specification.
+
+    * **Core features**:
+
+        * Write NWB:N HDF5 files
+        * Read NWB:N HDF5 files
+        * Read/Write data for custom extensions
+
+    * **Main contributors for this release:**
+        * Nathan Clack and Lawrence Niu (Vidriotech) : Software design and development lead
+        * Karel Svoboda : Project Lead
+        * Andrew Tritt and Oliver Ruebel : Review and discussion
+
+
+Open GitHub Issues for this release
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **PyNWB:** https://github.com/NeurodataWithoutBorders/pynwb/issues?q=is%3Aopen+is%3Aissue+milestone%3Asfn_release
+* **nwb-schema:** https://github.com/NeurodataWithoutBorders/nwb-schema/issues?q=is%3Aopen+is%3Aissue+milestone%3Asfn_release
+
+
+.. _dev-nwb2:
+
+NWB:N 2.0 (~June 2018)
+----------------------
+
+The primary purpose of this targe release is to stabilize and complete development of NWB:N 2.0beta. This release will also include a number of changes to the format and APIs that have not made it into the beta release. Request for new features for this release has been frozen. Additional new features may be included in future NWB:N 2.x releases.
+
+Planed new features (beyond NWB:N 2.0 beta)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **NWB:N Format Specification (and PyNWB API)**
+
+    * Support saving of NWB:N specifications in NWB:N HDF5 files as part of the ``/general/specifications``. Support use of these cached specification for data read.
+    * Reorganize ``/general`` to ease additiona and management of custom, lab-specific metadata via extensions
+
+* **NWB:N Specification Language (and PyNWB API)**
+
+*   * No major new features are planned for the specification language itself that have not been added in NWB:N 2.0 beta.
+
+* **PyNWB**
+
+    * Expand legacy read support to more sets of Allen data
+    * Expand and refine existing features for read/write/specfication etc.
```

Comment:
specification

Reply:
@mgrauer What change does this comment refer to?

Reply:
sorry @oruebel 

s/specfication etc./specification etc./

Reply:
Thanks @mgrauer. Fixed now.